base_comment_template/models/base_comment_template.py

```diff
@@ -15,69 +16,47 @@ class CommentTemplate(models.AbstractModel):
         "base.comment.template to put header and footer "
         "in reports based on created comment templates"
     )
+    _comment_template_partner_field_name = "partner_id"
```

Comment:
I don't really understand the need for this variable... Could you explain a bit, maybe through a comment?

Reply:
This should have a comment indeed for explaining the purpose, but I tell you quickly: it's for allowing to have another field that determines the source partner for the comment templates. Imagine other models where the partner field is called `customer_id` for example...

Reply:
Ok, I see... :+1: Thanks